extensions/wikia/TrackingOptIn/js/trackingOptInProxy.js

```diff
@@ -41,20 +41,29 @@ define('wikia.trackingOptIn', [
 		return win.M.trackingQueue;
 	}
 
+	function isOptedIn() {
+		log(['isOptedIn - proxy', optIn], log.levels.info, logGroup);
+		return optIn;
+	}
+
 	function pushToUserConsentQueue(callback) {
 		queue.push(callback);
 	}
 
-	function isOptedIn() {
-		log(['isOptedIn - proxy', optIn], log.levels.info, logGroup);
-		return optIn;
+	function geoRequiresTrackingConsent() {
+		if (win.M === 'undefined' || typeof win.M.continent === 'undefined') {
+			return true;
+		}
+
+		return win.M.continent === 'EU';
```

Comment:
Is there any chance to get real boolean from trackingOptIn module loaded in mobile-wiki?

Reply:
Done.